Cargo.toml

```
@@ -3,44 +3,26 @@ members = ["crates/*"]
 resolver = "2"
 
 [workspace.package]
+version = "0.0.1"
 rust-version = "1.70"
 edition = "2021"
-license = "MIT OR Apache-2.0"
+license = "Apache-2.0"
 authors = ["OpenQASM3 parser team"]
-
-[profile.dev]
-# Disabling debug info speeds up builds a bunch,
-# and we don't rely on it for debugging that much.
-# debug = 0
-
-[profile.dev.package]
-# These speed up local tests.
-# rowan.opt-level = 3
-# rustc-hash.opt-level = 3
-# smol_str.opt-level = 3
-# text-size.opt-level = 3
-# # This speeds up `cargo xtask dist`.
-# miniz_oxide.opt-level = 3
-
-# [profile.release]
-# incremental = true
-# # Set this to 1 or 2 to get more useful backtraces in debugger.
-# debug = 0
+readme = "README.md"
+keywords = ["QASM", "OpenQASM3", "parser"]
```

```suggestion
keywords = ["QASM", "OpenQASM 3", "parser"]
```
if we're allowed spaces

---

I'd have to check if cargo allows them or not, but I think on crates.io it's normalized to all lower case and with `-` only https://crates.io/keywords

---

Ah yeah, looks like it.  `openqasm3` is fine by me - it's mostly `OpenQASM3` with the mix of partially using the preferred stylisation but with a lack of space that jars me.

---

I switched it to openqasm3 in https://github.com/Qiskit/openqasm3_parser/pull/13/commits/ad75fca8e31c4b4188129a1b880fe3696726dc0a